addons/chemical/functions/fnc_changeGasMaskFilter.sqf

```diff
@@ -0,0 +1,24 @@
+#include "script_component.hpp"
+/*
+ * Author: DiGii
+ * 
+ * Arguments:
+ * 0: Target <OBJECT>
```

Comment:
player?

Reply:
no cause you can do it on your self or others